rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,200 @@
+
+    In works we use digital locations in `work.items.locations` to link to external resources.
+    We don't want to reuse that structure, which is more complicated than we need here.
+
+    Q: Aside from the URL, what other values do we want for these links?
+    e.g. link label
+
+    Q: Do we need these values at all?
+    If we put the Wikidata ID in the list of identifiers, do we leave clients to construct a URL, or do we provide it in the API?
+
+-   The `date`/`place` information is somewhat speculative.
+
+    We think we might include birth/death information on per-person pages; we could also link to significant dates and places and use that as a way to link between concepts.
+    This is similar to production events on works.
+
+    We can start with a `birthDate/birthPlace/deathDate/deathPlace` field on the initial version of the API, and decide if we want to put this information on per-person pages.
+    If yes, we can think about how to model this properly.
+    If no, we can scrap all of these fields.
+
+-   The `connectedConcepts` field lets us link between concept pages.
+    It contains a simplified subset of a Concept's fields; enough to render a link but nothing more.
+
+    Q: Do we want the connections between concepts to be discoverable from a single concept endpoint, or do we want that elsewhere?
+    I don't think we need to decide exactly how we'll model it (yet), but we should decide if it lives here or in at a distinct endpoint.
+
+    We don't want to describe how two concepts are connected; this is significantly more complicated.
+
+    Q: Is connectedness/relatedness symmetric?
+    If `A.connectedConcepts = [B]`, is it true that `B.connectedConcepts = [A]`?
+    This might affect if/how we build an API for listing concepts.
+
+
+
+## Endpoint #2: Concepts in the works API
+
+We'll simplify how concepts appear in the works API.
+We'll provide enough information to render a link to the per-person or per-subject page, but nothing more -- for detailed information, use the concepts API.
+
+For example:
+
+```
+GET /works?contributors.agent=azxzhnuh
+{
+  …,
+  "contributors": [
+    {
+      "agent": {
+        "id": "azxzhnuh",
+        "label": "Florence Nightingale",
+        "type": "Person"
+      },
+      "roles": [
+        {
+          "label": "Author",
+          "type": "ContributionRole"
+        }
+        ...
+      ]
+    }
+  ]
+}
+```
+
+
+
+## Endpoint #3: Listing and searching concepts
+
+If we decide we need this, it would look similar to the works and images endpoints:
+
+```
+GET /concepts
+{
+  "type": "ResultList",
+  "results": [...]
+}
+```
+
+The `results` list would contain the same objects as returned from endpoint #1.
+
+Filtering, querying, aggregations, and so on, would follow the same patterns as the works and images APIs.
+We don't need to decide the details of this yet.
+
+Q: Do we need this endpoint?
+Is listing/searching concepts something we think we'll do?
```

Comment:
Yes, though maybe not initially. We will likely ultimately end up using it for both search and building out a browse interface to subjects and people.